x-pack/plugins/apm/public/components/shared/charts/Timeline/Marker/error_marker.tsx

```diff
@@ -108,6 +108,7 @@ export function ErrorMarker({ mark }: Props) {
           key={mark.serviceColor}
           color={mark.serviceColor}
           text={error.service.name}
+          indicator={() => <span />}
```

Comment:
Would `null` work also?

Reply:
Any reason why `indicator` is a function in the first place?

```suggestion
          indicator={false}
```

Reply:
The Legend will show the default indicator (the dot) if it's false or null

```javascript
export function Legend({
  onClick,
  text,
  color,
  disabled = false,
  clickable = false,
  shape = Shape.circle,
  indicator,
  ...rest
}: Props) {
  const theme = useTheme();
  const indicatorColor = color || theme.eui.euiColorVis1;

  return (
    <Container
      onClick={onClick}
      disabled={disabled}
      clickable={clickable || Boolean(onClick)}
      {...rest}
    >
      {indicator ? (
        indicator()
      ) : (
        <Indicator color={indicatorColor} shape={shape} withMargin={!!text} />
      )}
      {text}
    </Container>
  );
}
```